### PR TITLE
Fix click-to-seek for audios which are not spanning the entire canvas

### PR DESCRIPTION
**Pull request by whenov**
Solves #686

### CODE REVIEW

File: src/wavesurfer.js [hunk 1]
```diff
@@ -92,6 +92,20 @@ var WaveSurfer = {
...
+            var parentWidth = my.drawer.getWidth();
+
+            if (!my.params.fillParent && nominalWidth < parentWidth) {
+                var bbox = my.drawer.wrapper.getBoundingClientRect();
+                progress = (e.clientX - bbox.left) * my.params.pixelRatio / nominalWidth;
+
+                if (progress > 1) {
+                    progress = 1;
+                }
+            }
```
Comment (katspaugh) [inline src/wavesurfer.js]: This logic should be inserted here: https://github.com/katspaugh/wavesurfer.js/blob/master/src/drawer.js#L44 or https://github.com/katspaugh/wavesurfer.js/blob/master/src/drawer.js#L63, in the drawer code.

Comment (whenov) [inline src/wavesurfer.js]: I was trying to do that, but getDuration() is not accessible in drawer

Comment (katspaugh) [inline src/wavesurfer.js]: True, but I think you can take `this.drawer.width` (or `this.width` in the drawer code) instead of calculating `nominalWidth`.

Comment (whenov) [inline src/wavesurfer.js]: Reasonable.

**Comment by katspaugh** [comment]
Nice, thanks!

**Pull request merged**